src/plugins/assert/assertHttpRegionParser.ts

```diff
@@ -69,8 +69,8 @@ export async function parseAssertLine(
         nextParserLine: next.value.line,
         symbols: [
           {
-            name: match.groups.key,
-            description: match.groups.value,
+            name: match.groups.key ?? "script",
+            description: match.groups.value ?? "script",
```

Comment:
match.groups.value seems to be a Copy Paste error of me. I would change it to this
```suggestion
            description: `${match.groups.predicate} ${match.groups.expected || ''}`.trim(),
```